changelog/12017.improvement.rst

```diff
@@ -0,0 +1,5 @@
+* migrate formatting tests to fstrings
```

Comment:
This does not feel like an improvement as seen by users, but an internal/trivial change. I would even consider not adding a changelog at all, but up to you. 👍